src/client/js/Controls/PropertyGrid/Widgets/styles/SvgSelectWidget.css

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
still having hard time with the scss, but as soon as they will be forced I will only use those.